AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/BundleHelper.java

```diff
@@ -172,6 +173,10 @@ public BundleHelper(AndroidAnnotationsEnvironment environment, TypeMirror elemen
 			if (isTypeParcelable(type)) {
 				methodNameToSave = "put" + "Parcelable";
 				methodNameToRestore = "get" + "Parcelable";
+			} else if (annotationHelper.isAnnotatedWith(type, "org.parceler.Parcel")) {
```

Comment:
I'm finding that matching types following the critieria we discussed may be a bit overzelous.   For instance, this is matching the `ArrayList<String> listExtra` in `ExtraInjectedActivity`.  Should we be serializing this through Parceler?